### PR TITLE
Expand on information

### PR DESCRIPTION
**Pull request by pplu**
I'd like to transmit more information in these three sections

### CODE REVIEW

File: actdocs/templates/index/whatis [hunk 1]
```diff
@@ -5,22 +5,23 @@
...
 	  </p>
     </div>
         
     <div class="col-sm-4 wow fadeInDown text-center">
       <img  class="rotate" src="img/icon/laptop.svg" alt="Generic placeholder image">
        <h3>Perl</h3>
        <p class="lead">
-		Yes! But not only Perl.
+		Yes! But not only Perl. Attendees and <a href="/barcelona2016/newtalk">speakers</a> from other communities are welcome. JavaScript, PHP, Ruby, Python, C, Java, Smalltalk, Linux, MacOS, Windows, Makers, Web Developers, Data Scientists, etc are invited. All sexes, sizes and ages. You only need to agree with our <a href="/barcelona2016/coc.html">CoC</a>.
```
Comment (meis) [inline actdocs/templates/index/whatis:16]: I know it's a complicated matter and maybe I'm getting the semantics wrong but I'm not sure if 'agree' is the right word here.

Personally, I'm not sure if I agree with the CoC (or with the fact that we have a CoC) and I don't think that it should be motive not to allow me in.

I'd probably say something like 'follow' or some other synonym.

**Comment by meis** [comment]
hey @pplu, thanks to collaborate!

I think all these new information is good and adds a lot of value. I'm not sure about the presentation though.

pr16

With those changes, the three columns look unbalanced. Maybe it's just me but I find that asymmetric white space annoying. I had the same problem when trying to fulfill the columns and that's why I ended up with the (probably insufficient) short sentences.

I'm not sure if it's better to leave the 3 sections short and add the relevant information to the paragraphs below or just try to balance the three columns a bit.

Any thoughts? It's ok if they include kittens..

File: actdocs/templates/index/whatis (hunk 1 above)
Comment (pplu) [reply actdocs/templates/index/whatis:16]: I like "follow" better too :smile:

**Comment by pplu** [comment]
> With those changes, the three columns look unbalanced. Maybe it's just me but I find that asymmetric white space annoying. I had the same problem when trying to fulfill the columns and that's why I ended up with the (probably insufficient) short sentences.

Jejeje... I don't find the whitespace specially annoying

> I'm not sure if it's better to leave the 3 sections short and add the relevant information to the paragraphs below or just try to balance the three columns a bit.

I wanted to catch their attention early on, without making them really have to read much more to make the decision to continue reading... I'm not sure how to balance it out...

How do you like this wording for the "Perl" section:

But not limited to Perl: we welcome attendees and speakers from any Profession or Programming language community. All sexes, sizes and ages. You only need to follow our CoC

Maybe having some some keywords in that sentence have a tooltip would work.
For "profession": Web Developers, Data Scientists, Makers, Biologists
For "Programming language": JavaScript, PHP, Ruby, Python, C, Java, Smalltalk...
For sexes: "Male, female, Gay, Lesbian, Trans"
For Ages: "0-122" (note that I've just looked up the longest know human lifespan)

Does it work for you?

**Comment by meis** [comment]
> Does it work for you?

Yes, i think that might work!

Once I've applied the changes I thought about more thinks I'd like to change. Let me apply it and I'll submit a PR to discuss :)

**Pull request merged**